### ISSUE TITLE
How to use it with python on windows10

### ISSUE THREAD

**Issue by kblade07** [ItWasAllIntended/Steam-Comment-Bot#1]
Hi. Im searching for a bot like this for over 2 months and I found this one today. Im used to bots made in js and running with nodeJS, not pyhton but Ive used python in the past once.
Im using it on windows, Ive added python to the PATH variables so i can use it with CMD, but when i run python SteamCommentBot.py STEAMID im getting 2 errors:
1) syntax error: invalid syntaxt stdin line 1
2) Module not found error, no module named "asks"

Im a noob in phython and i dont know how to troubleshoot this and how to start the script...Can you please explain me what I need to do? Ive tried to import the 2 modules in the req txt but i dont really know how...
Thank you very much

**Comment by ItWasAllIntended** [comment]
If you've added Python to your PATH you should also have Pip (The Python package installer on your path). Navigate to the repo dir with cmd and run `pip install -r requirements.txt`. Run the script again and it should work

**Comment by kblade07** [comment]
Thank you very much, it worked. 
One more question, can I change the amout of comments the accounts is posting? Like 20 comments.
And can I change so the account will post different texts imported from the .txt file? Currently the script is posting 1 comment and 1 final comment

**Comment by ItWasAllIntended** [comment]
Edit the text file and everything will sort itself out. It'll only post as many many comments as it can divide up from the file

**Comment by ItWasAllIntended** [comment]
I'm very sorry I just understood what you really asked. There isn't a way currently to force a comment to be split but I can add that in when I get a chance. Currently it will only split the comments so none of them exceeds the 1000 character comment limit

**Comment by kblade07** [comment]
Hey, thanks for the reply. Yea this is what im talking about. I did something in the code, about the length. Instead of 999, I switched to 5 and I added random texts like above. It is posting multiple comments, splitting them but if I add 5 lines, the script is posting 13 comments and only 5+final comment is posted. So yea its pretty confusing for the script if I change the length of the line. But its a good solution for now :D
